megazords/ios-rust/ios/Generated/glean.swift

```diff
@@ -0,0 +1,4159 @@
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
```

Comment:
Why do we need this file? Shouldn't this be generated by `uniffi` on `rust-components-swift`? I don't mind it generated here, but we'll need some consistent script that runs to generated updates.


(if we need it here, we can start a discussion around generally checking-in generated uniffi files, which TBH I'm in support of, but it seems beyond the scope of this PR)

Reply:
This shouldn't be here, I think it's a side effect of me playing around with the TestServices project trying to get it to work.  Let me clean that up...